### PR TITLE
adding function to create vector mask from shape

### PR DESCRIPTION
**Pull request by pineapplespatula**
Added new path based adapter functions, and bumped the version number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "spaces-adapter",
-  "version": "0.32.1",
+  "version": "0.33.0",
```
Comment (iwehrman) [inline package.json]: This should be removed. The current best workflow is to increment the version number after the PR is merged using `npm version minor`:

1. wait for PR to be merged.
2. update local branch: `git pull master` 
3. increment minor version, creating a new commit and a new tag `npm version minor`
4. push updated branch and tag: `git push origin --tags master`

**Comment by iwehrman** [comment]
Also, there are typos in the JSDoc.

File: package.json (hunk 1 above)
Comment (pineapplespatula) [inline package.json]: is that just so that we tag the branch with master? is that the only difference between the two workflows?

**Comment by pineapplespatula** [comment]
@iwehrman :repeat:  I'm still not clear on why the versioning shouldn't be bumped in the same PR. it seems pedantic to me

**Comment by iwehrman** [comment]
The reason is that we need to tag versions with git, and we can't do that in the branch because commits tend to change during PRs. Also, if you bumped the version in the child commit in your branch, what we would really want to do is tag the merge commit instead of the child commit. So, it's just easier overall to do both the versioning and the tagging at the same time. Also, if there are two open PRs at the same time then they won't conflict with each other.

**Comment by iwehrman** [comment]
The JSDoc syntax is still incorrect. Back to @pineapplespatula.

**Comment by iwehrman** [comment]
Looks good, thanks!

**Pull request merged**